openstack/main.tf

```diff
@@ -77,6 +77,10 @@ variable node_assign_floating_ip {
   default = "false"
 }
 
+variable node_extra_disk_size {
```

Comment:
I think also `node_extra_disk_size` should default to 0
and then we need to add under the node section:
```# Disk settings
 extra_disk_size = "${var.node_extra_disk_size}"```